common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -144,6 +143,7 @@ public AzureActiveDirectoryWebViewClient(@NonNull final Activity activity,
         mRedirectUrl = redirectUrl;
         mCertBasedAuthFactory = new CertBasedAuthFactory(activity);
         mSwitchBrowserRequestHandler = switchBrowserRequestHandler;
+        mIsRunningInAuthProcess = ProcessUtil.isRunningOnAuthService(getActivity().getApplicationContext());
```

Comment:
> mIsRunningInAuthProcess = ProcessUtil.isRunningOnAuthService(getActivity().getApplicationContext()); [](http://example.com/codeflow?start=7&length=101)

This can all be in OAuth2WebViewClient.

Reply:
Losing my mind. I thought i had checked this and it wouldn't let me, but definitely me missing something. moved it